.github/workflows/build-firmware.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
         # see also build-primary-bundle where default/primary bundle is build separately
         #
         build-target: [
-            all,
+            f407-discovery,
```

Comment:
would this add second f407-discovery?

Reply:
That change isn't there anymore, I'm not sure when this review comes from.

Reply:
i am reviewing commits one after another. if commit history is misleading can we please squash?